web-app/src/main/webapp/index.html

```diff
@@ -19,8 +20,67 @@
         </div>
       </form>
     </div>
+    <!-- Map -->
     <div id="map">
       A map will appear here
     </div>
+    <!-- Report submission form -->
+    <div class="container">
+      <form id="report-form">
+        <div class="row">
+          <div class="two-thirds columns">
+            <input class="u-full-width" type="text" placeholder="Title"
+              id="title-input">
+          </div>
+        </div>
+        <div class="row">
+          <div class="seven columns">
+            <input class="u-full-width" type="text" placeholder="Location"
+              id="location-input">
+          </div>
+          <div class="one column">
+            <img src="images/map.svg" alt="button to pick location using map">
+          </div>
+        </div>
+        <div class="row">
+          <div class="two-thirds columns">
+            <input class="u-full-width" type="datetime-local" placeholder="Time"
+              id="time-input">
+          </div>
+        </div>
+        <div class="row">
+          <div class="two-thirds columns">
+            <select class="u-full-width" type="text" placeholder="Category"
+              id="category-input">
+              <option value="foo">foo</option>
+              <option value="bar">bar</option>
+            </select>
+          </div>
+        </div>
+        <div class="row">
+          <div class="two-thirds columns">
+            <input class="u-full-width" type="text" placeholder="Description"
+              id="description-input">
+          </div>
+        </div>
+        <div class="row">
+          <div class="two-thirds columns">
+            <button class="u-full-width" type="button"
+              id="attach-image-button">Attach an image (optional)</button>
+          </div>
+        </div>
+        <div class="row">
+          <div class="two-thirds columns">
+            <button class="u-full-width" type="submit"
+              id="submit-button">Submit</button>
+          </div>
+        </div>
+      </form>
+    </div>
+    <!-- Bottom dock -->
+    <div class="container u-full-width" id="dock">
+    </div>
+    <img id="report-button" src="images/report.svg" alt="button to report a new
```

Comment:
nit: shouldn't the report-button be inside the bottom dock container?

Reply:
Fixed this in a later PR!